src/rabbit_plugins.erl

```diff
@@ -211,6 +211,12 @@ is_loadable(App) ->
         _                            -> false
     end.
 
+
+%% List running plugins along with their version.
+-spec running_plugins() -> [{atom(), Vsn :: string()}].
+running_plugins() ->
+    {ok, [{App, Vsn} || {App, _ , Vsn} <- rabbit_misc:which_applications(), lists:member(App, active())]}.
```

Comment:
This is hardly a serious concern for this function given that it will be used only by CLI tools but I'd cache the return value of `active/0` instead of calling it N times.